testsuite/features/support/twopence_init.rb

```diff
@@ -109,7 +109,9 @@ def twopence_init(target)
   $rhlike_minion = twopence_init("ssh:#{ENV['RHLIKE_MINION']}") if ENV['RHLIKE_MINION']
   $deblike_minion = twopence_init("ssh:#{ENV['DEBLIKE_MINION']}") if ENV['DEBLIKE_MINION']
   $build_host = twopence_init("ssh:#{ENV['BUILD_HOST']}") if ENV['BUILD_HOST']
-  $nodes += [$minion, $ssh_minion, $rhlike_minion, $deblike_minion, $build_host]
+  $sles_salt_migration = twopence_init("ssh:#{ENV['SLES_SALT_MIGRATION']}") if ENV['SLES_SALT_MIGRATION']
+  $leap_salt_migration = twopence_init("ssh:#{ENV['LEAP_SALT_MIGRATION']}") if ENV['LEAP_SALT_MIGRATION']
```

Comment:
We will never have both SLES_SALT_MIGRATION and LEAP_SALT_MIGRATION coexisting at same time.

Wouldn't be much more simpler to use SALT_MIGRATION_MINION as environment variable, and depending the environment you set one value or another?

Reply:
This makes sense. Thank you for the suggestions.